core/templates/dev/head/pages/exploration_editor/ExplorationEditorFeaturesBackendApiService.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ *  @fileoverview Service for fetching the features the exploration editor is
+ *  configured to support.
+ */
+
+oppia.constant(
+  'EXPLORATION_EDITOR_FEATURES_URL',
+  '/exploration_editor/features/<exploration_id>');
+
+oppia.factory('ExplorationEditorFeaturesBackendApiService', [
+  '$http', 'UrlInterpolationService', 'EXPLORATION_EDITOR_FEATURES_URL',
+  function($http, UrlInterpolationService, EXPLORATION_EDITOR_FEATURES_URL) {
+    return {
+      fetchExplorationEditorFeatures: function(explorationId) {
```

Comment:
Can you add some documentation to this method? It might help if the callers knew, for instance, which permissions are needed in order to access this data. That helps avoid taking the trip to the backend in order to determine that.